src/store/ChartState.js

```diff
@@ -20,6 +24,9 @@ class ChartState {
         this.mainStore = mainStore;
         this.chartStore = mainStore.chart;
         when(() => this.context, this.onContextReady);
+        reaction(() => this.onExportLayout, this.ExportLayout.bind(this));
```

Comment:
@MahboobehMohammadi You can call `ExportLayout`, `ImportLayout` and `CleanChart` in `updateProps` function. It will be invoked whenever one of the props is changed.